src/Symfony/Component/Form/Tests/ResolvedFormTypeTest.php

```diff
@@ -132,10 +127,6 @@ public function testCreateBuilder()
 
     public function testCreateBuilderWithDataClassOption()
     {
-        if (version_compare(\PHPUnit_Runner_Version::id(), '3.7', '<')) {
-            $this->markTestSkipped('This test requires PHPUnit 3.7.');
-        }
```

Comment:
@stof I just submitted #13303 with the minor change. Once this is merged into 2.5, 2.6 and 2.7 I can rebase this PR. Anything else to do? /cc @fabpot